adapters/v1/backend_utils.go

```diff
@@ -62,6 +63,13 @@ func (a *BackendAdapter) postResultsAsGoroutine(ctx context.Context, report *v1.
 	}(report, eventReceiverURL, imagetag, wlid, errorChan, wg)
 }
 
+func (a *BackendAdapter) getRequestHeaders() map[string]string {
+	return map[string]string{
+		"Content-Type":              "application/json",
+		beServer.RequestTokenHeader: a.accessToken,
```

Comment:
As I wrote in the `backend` PR, we should add the header only if the string is not empty. (unless an empty header does not affect us)

Reply:
Empty header does not affect